test/Serilog.Formatting.Compact.Tests/Serilog.Formatting.Compact.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Could also track Serilog.Tests, here:

```xml
    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462;net48</TargetFrameworks>
    <TargetFrameworks>$(TargetFrameworks);net7.0;net6.0;net5.0;netcoreapp3.1;</TargetFrameworks>
```

Reply:
I'd skip net5.0, as this is no more supported, but for test maybe good to have it still there?